test_hello_worlds/python.cpp

```diff
@@ -0,0 +1 @@
+print("This line will be printed.")
```

Comment:
This file contains at least one console log. Please remove any present. - additions.length